util/src/main/java/io/kubernetes/client/util/ClientBuilder.java

```diff
@@ -100,28 +99,46 @@ public static ClientBuilder standard() throws IOException {
     return standard(true);
   }
 
+  public static ClientBuilder context(String context) throws IOException {
+    final File kubeConfig = getKubeConfigFile();
```

Comment:
This isn't quite what I was expecting, I think that we should support `ClientBuilder.standard().context("foo").build()` as well as `ClientBuilder.kubeconfig(someFile).context("foo").build()`

I think what this means is that we need to save off the KubeConfig object in the builder rather than constructing it new each time.